README.md

```diff
@@ -1,21 +1,13 @@
 # Gravitas
 
-**TODO: Add description**
+An agent to control your infrastructure.
 
-## Installation
+## Values
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `gravitas` to your list of dependencies in `mix.exs`:
-
-```elixir
-def deps do
-  [
-    {:gravitas, "~> 0.1.0"}
-  ]
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/gravitas](https://hexdocs.pm/gravitas).
+The values that guide the design of Gravitas are:
 
+- *Observability*: You should be able to understand what Gravitas is doing easily. Users should be able to maintain an understanding of the problem from Gravitas' perspective. The user should be able to have a representation of the problem state shared with Gravitas and a representation of both the activities of Gravitas and other users using it. Users need to be able to see what Gravitas is doing and what it will do next relative to its state. And this without need for overly taxing cognitive activity
```

Comment:
This is really verbose, i think we should cut down, but i am not sure of what to keep and what to throw away yet.

Reply:
Maybe try something like `The state and actions of Gravitas should be visible, cause and consequent action taken should be visible and *understandable* to a user.`